### PR TITLE
fix: Fix initial glitch (Context solution)

### PR DESCRIPTION
**Pull request by alexanderGugel**
#206

### CODE REVIEW

File: renderers/Compositor.js [hunk 1]
```diff
@@ -243,6 +243,10 @@ Compositor.prototype.receiveCommands = function receiveCommands(commands) {
     for (var i = 0; i < len; i++) {
         this._inCommands.push(commands[i]);
     }
+    
+    for (var selector in this._contexts) {
```
Comment (michaelobriena) [inline renderers/Compositor.js]: Would really prefer if this was an associated array.  This gets called a lot.

Comment (alexanderGugel) [inline renderers/Compositor.js]: I agree, there is actually a TODO in the code, but I don't think this would belong in this PR. Should I change it?

Comment (michaelobriena) [inline renderers/Compositor.js]: We can merge this then change.

**Comment by michaelobriena** [comment]
landed as 90f5784

**Comment by alexanderGugel** [comment]
This doesn't work for nested scenes because of #190

**Pull request closed without merging**